tests/TestLaunchExporter.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import os
+import unittest
+
+sys.path.append('.')
+from unittest import TestCase
+from vrops_exporter import parse_params
+
+
+class TestLaunchExporter(TestCase):
+
+    def test_with_cli_params_1(self):
+        sys.argv = ['prog', '-u', 'testuser', '-p', 'testpw31!', '-o', '1234']
```

Comment:
please add debug

Reply:
test with debug option on

Reply:
in this test i set the option and checked that the option debug is set.